contrib/freshmaker-cli

```diff
@@ -50,6 +55,7 @@ EVENT_TYPES = {
     12: {"name": "ODCS compose state changed", "url": ""},
     13: {"name": "Manual rebuild with advisory",
          "url": "https://errata.devel.redhat.com/advisory/{search_key}"},
+    100: {"name": "No event", "url": ""}
```

Comment:
This looks like a fake event type. How about making the key `None`? Then you can set `"event_type_id": 100,` to `"event_type_id": None,` which seems more natural.